urdf/include/urdf/model.h

```diff
@@ -58,6 +59,8 @@ class Model: public ModelInterface
   bool initFile(const std::string& filename);
   /// \brief Load Model given the name of a parameter on the parameter server
   bool initParam(const std::string& param);
+  /// \brief Load Model given the name of a parameter on the parameter server using provided nodehandle
+  bool initParamWithNodeHandle(const std::string& param, const ros::NodeHandle& nh = ros::NodeHandle());
```

Comment:
Adding this looks good to me. It makes using the library easier.

I think the specific case of ros-controls/ros_controllers#244 could be solved by using [NodeHandle::resolveName("robot_description")](http://docs.ros.org/kinetic/api/roscpp/html/classros_1_1NodeHandle.html#a9eab8c5ff7ca9d3468d2a35dcd84ae40) and passing the fully qualified name into initParam()